src/main/java/database/service/JsonDatabaseService.java

```diff
@@ -245,8 +246,9 @@ String getDatabasePath(Class<? extends BaseEntity> entityClass) {
     }
 
     private <T extends BaseEntity> void updateEntityFields(T outcomeEntity, T incomeEntity) {
```

Comment:
I think the same changes should be pplied for `getByFilters` method. Now it searchs only in class fields, but not in supercalsses. Please check and fix if needed. And don't forget to provide full list of fields to `Validator.validateDatabaseFilters` in `getByFilters`.